### ISSUE TITLE
There is a question at startup

### ISSUE THREAD

**Issue by g558800** [bmaltais/kohya_ss#731]
Validating that requirements are satisfied.
All requirements satisfied.
headless: False
Load CSS...
Running on local URL:  http://127.0.0.1:7860

What is the mean: "headless: False"

**Comment by wcole3** [comment]
Headless means that the app is running without a GUI.  Here if even if you add the `--headless` cmdline argument you can still open the webserver gui, but I believe it was added to support people using services like vast.ai ([see commit](https://github.com/bmaltais/kohya_ss/commit/103a9919c7fe681e7ffd2c18b7a07ab9a736c14d))

**Comment by bmaltais** [comment]
This is correct. It will remove all but one file and folder buttons to prevent TK library interactions on systems with no local display like runpod or vast.ai
